compiler/luci/pass/src/RemoveRedundantTranspose.cpp

```diff
@@ -61,9 +59,9 @@ bool remove_consecutive_transpose_function(luci::CircleNode *node)
   }
   else
   {
-    auto g = main_perm->graph();
-    auto new_const_node = g->nodes()->create<luci::CircleConst>();
+    auto g = main_node->graph();
 
+    auto new_const_node = g->nodes()->create<luci::CircleConst>();
```

Comment:
This change seems to be a format change. It would be better to separate this.

Reply:
okay. I make a roll-back to original.